torch/csrc/jit/codegen/cuda/test/test_gpu3.cpp

```diff
@@ -6167,6 +5891,33 @@ TEST_F(NVFuserTest, FusionNullScheduler3_CUDA) {
   }
 }
 
+TEST_F(NVFuserTest, FusionReducingZeroElements_CUDA) {
+  GTEST_SKIP()
+      << "Merging IterDomains with ending values that are 0 is not supported at this time.";
```

Comment:
What do you mean? What's missing?

Reply:
The text here is just a copy-paste of the error message I see. I guess it is because the reduction scheduler tries to merge an ID of extent 0 with another ID

Reply:
OK. If we need to support this, please also create an issue.

Reply:
I already left a comment in the code:
```cpp
  // TODO: It would be nice if in this function we could convert reduction of
  // zero elements into a `FullOp`
```
I will work on that as soon as this PR is merged. If it eventually turns out to be too hard, then I will create an issue tracking it.